.github/workflows/draft-release.yml

```diff
@@ -19,53 +19,53 @@ jobs:
   build:
     if: contains(fromJson('["webrtc", "react-client"]'), github.event.inputs.package)
 
-    name: '@telnyx/${{ github.event.inputs.package }} Draft release'
+    name: "@telnyx/${{ github.event.inputs.package }} Draft release"
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        # https://github.com/peter-evans/create-pull-request/issues/122
-        ref: 'main'
-        # https://github.com/release-it/release-it/issues/657#issuecomment-647848200
-        fetch-depth: 0
-    - name: Setup Git user
-      # https://github.com/actions/checkout/issues/13
-      run: |
-        git config user.name github-actions[bot]
-        git config user.email 41898282+github-actions[bot]@users.noreply.github.com
-    - name: Use Node.js 11.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: '11.x'
-    - name: Install shared dependencies
-      run: npm ci
-    - name: Install package dependencies
-      working-directory: packages/${{ env.PACKAGE_DIR }}
-      run: npm ci
-    - name: Test
-      working-directory: packages/${{ env.PACKAGE_DIR }}
-      run: npm test
-      env:
-        CI: true
-    - if: github.event.inputs.package == 'webrtc'
-      name: Generate docs
-      working-directory: packages/${{ env.PACKAGE_DIR }}
-      run: |
-        npm run docs # IDEA Set --gitRevision to release-it tag
-        git add docs
-        git commit -m "docs: Update TS docs" || echo "No docs changes to commit"
-    - name: Create draft release
-      working-directory: packages/${{ env.PACKAGE_DIR }}
-      run: |
-        npm run release -- \
-        ${{ github.event.inputs.version }} \
-        --ci \
-        --no-git.tag \
-        --github.draft \
-        --no-npm.publish \
-        --no-npm.tag
-      env:
-        NPM_TOKEN: ${{ secrets.NPM_CI_TOKEN }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+          # https://github.com/peter-evans/create-pull-request/issues/122
+          ref: "main"
+          # https://github.com/release-it/release-it/issues/657#issuecomment-647848200
+          fetch-depth: 0
+      - name: Setup Git user
+        # https://github.com/actions/checkout/issues/13
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email ${{ secrets.GIT_REPO_USERNAME }}
```

Comment:
This email is a default one provided by GitHub actions bot and it is used to commit in git with the author as `git-action-bot`

https://github.community/t/github-actions-bot-email-address/17204/4

Reply:
I tried to use another username and password provided here, but the icon get grey, but it is working

![image](https://user-images.githubusercontent.com/16343871/144089124-db5bd3b9-cee8-4d31-9619-067adba550a3.png)

We can use this if you prefer @arghav 
https://github.com/actions/checkout/blob/main/README.md#push-a-commit-using-the-built-in-token